services/service.js

```diff
@@ -238,7 +251,8 @@ service.register("getAppInfo", async (message) => {
       throw new Error('Fail to parse json');
     }
 
-    const appList = await promiseCall(service, 'luna://com.webos.applicationManager/dev/listApps', {});
+    const svc = installerService();
+    const appList = await promiseCall(svc, 'luna://com.webos.applicationManager/dev/listApps', {});
```

Comment:
Forgot about that - `service.call` (used by `promiseCall`) is not implemented by `ServiceRemote` yet.

Reply:
Implemented now :)